src/main/java/fact/filter/MovingMinimum.java

```diff
@@ -0,0 +1,111 @@
+package fact.filter;
+
+import fact.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import stream.Data;
+import stream.ProcessContext;
+import stream.StatefulProcessor;
+import stream.annotations.Description;
+import stream.annotations.Parameter;
+
+/**
+ * 
```

Comment:
I think this is the right place for some general information about what the processor does in a few short words.